README.md

```diff
@@ -35,7 +35,7 @@ Environments that should work include:
 
 or installed by cloning the GitHub repo and running `make install`:
 
-    $ git clone git://github.com/aheckmann/m.git && cd m && make install
+    $ git clone git@github.com:aheckmann/m.git && cd m && make install
```

Comment:
Web clone url as a default works for everyone including anon users. I believe the `ssh` url requires setting up keypair and enabling `ssh` access.